dbt_subprojects/daily_spellbook/models/polymarket/polygon/polymarket_polygon_positions_raw.sql

```diff
@@ -0,0 +1,70 @@
+{{
+  config(
+    schema = 'polymarket_polygon',
+    alias = 'positions_raw',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
```

Comment:
seeing that the incremental run runs for a long time, do we also switch this to `delete+insert`? 

also noticing that this doesn't have a partition keys specified, maybe that would help?

Reply:
i think this was isolated to a recent run, since it is running at the same time as your large solana PR. all PRs share the same cluster, so it's getting overloaded today.

you can see previous run today was more acceptable:
https://github.com/duneanalytics/spellbook/actions/runs/11553188411/job/32153823614

Reply:
with that in mind, i did still add partitions to the large raw tables at least